src/propertiesdialog.cpp

```diff
@@ -423,9 +436,12 @@ void PropertiesDialog::setupShortcuts()
 
         const auto shortcuts = keyAction->shortcuts();
         for (const QKeySequence &shortcut : shortcuts)
-            sequenceStrings.append(shortcut.toString());
+            sequenceStrings.append(shortcut.toString(QKeySequence::NativeText));
 
-        QTableWidgetItem *itemName = new QTableWidgetItem( tr(keyValue.toStdString().c_str()) );
+        QString txt = keyAction->text();
+        txt.remove(QRegularExpression(QStringLiteral("\\s*\\(&[a-zA-Z0-9]\\)\\s*")));
+        txt.remove(QLatin1Char('&'));
```

Comment:
Could you extract these lines to a function? The complex regular expression may be changed in the future and I'd like to manage them in one place.

Reply:
Will do it.